geo/Cargo.toml

```diff
@@ -16,11 +16,11 @@ travis-ci = { repository = "georust/geo" }
 [dependencies]
 num-traits = "0.2"
 serde = { version = "1.0", optional = true, features = ["derive"] }
-spade = "1.5.1"
+rstar = { version = "0.1" }
 failure = "0.1.2"
 postgis = { version = "0.6", optional = true }
 proj = { version = "0.5", optional = true }
-geo-types = { version = "0.2", features = ["spade"] }
+geo-types = { path = "../geo-types", features = ["rstar"] }
```

Comment:
This will need to be updated in order to publish the crate... feel free to change it to any version you see fit.

Reply:
yep, this makes sense 👍